e2e/test/MethodE2EMultiplexingTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    [TestClass]
+    [TestCategory("IoTHub-E2E")]
+    [TestCategory("ConnectionPooling_AMQP_E2E")]
```

Comment:
Please be consistent in naming - test categories are per service and will indicate if different prerequisites exist. 
This test is simply another aspect of the IoTHub-E2E tests.